examples/shared_samples/ethernet_network.rb

```diff
@@ -14,12 +14,6 @@
 # Example: Create/Update/Delete ethernet networks
 # NOTE: This will create an ethernet network named 'OneViewSDK Test Vlan', update it and then delete it.
 #   It will create a bulk of ethernet networks and then delete them.
-#
-# Supported APIs:
```

Comment:
any reason for removing these lines..?

Reply:
We are anyways giving this info from README and we need to edit this line for every release which seems to be not so useful to me.